doc/us-en/setup/Install-OpenEdge-on-Linux.md

```diff
@@ -1,2 +1,161 @@
+# Install OpenEdge on Linux
 
-> TO DO(@sherlockblaze)
\ No newline at end of file
+> OpenEdge is mainly developed in Go programming language and supports two startup modes: ***docker*** container mode and ***native*** process mode.
+
+This document focuses on the installation and configuration of the environment required for OpenEdge and the rapid deployment of OpenEdge on the Linux-like system.
+
+## Environment Configuration
+
+### Install Docker
+
+> OpenEdge offers two startup modes. To start using ***docker*** container mode (recommended), you need to complete the docker installation first.
+
+**Notice:**
+
++ The official Dockerfile is offered for multi-stage builds. If you need to build the relevant image yourself, The version of docker you installed should be above 17.05.
++ The production environment can run the image using a lower version of Docker, which is currently tested to a minimum usable version of 12.0.
++ According to the [Official Release Log](https://docs.docker.com/engine/release-notes/#18092), the version of docker version < 18.09.2 has some security implications. It is recommended to install/update the Docker version to 18.09. 2 and above.
+
+Can be installed by the following command(Suitable for linux-like systems, [Supported Platforms](./Support-platforms.md)):
+
+```shell
+curl -sSL https://get.docker.com | sh
+```
+
+#### Ubuntu
+
+Command:
+
+```shell
+sudo snap install docker # Ubuntu16.04 after
+sudo apt install docker.io # Ubuntu 16.04 before
+```
+
+#### CentOS
+
+Command:
+
+```shell
+yum install docker
+```
+
+***Notice***: 
+
++ After the Docker installation is complete, use the following command to view the installed Docker version.
+
+```shell
+docker version
+```
+
+#### Debian 9/Raspberry Pi 3
+
+Command:
+
+```shell
+curl -sSL https://get.docker.com | sh
+```
+
+**For more details, please see the [official documentation](https://docs.docker.com/install/).**
+
+### Install Python2.7 and Python runtime dependency package
+
+> + OpenEdge provides Python Runtime, which supports running code written in Python version 2.7. If you plan to start using ***native*** process mode, you need to install Python 2.7 and the package you depend on. If you plan to start in ***docker*** container mode, you do not need to perform the following steps.
```

Comment:
OpenEdge provides Python2.7 runtime, which supports developers write scripts in Python2.7 directly. If you run OpenEdge in **native process mode**, you **MUST** firstly install Python2.7 and the package actually use. But, if you run OpenEdge in **docker container mode**, you do not need to perform the following steps.